### PR TITLE
docs - first HA updates that use WAL replication.

### PR DESCRIPTION
**Pull request by mkiyama**
These are changes for the HA "how to" (procedural) topics in the Admin Guide. 
--Removed references to filerep
--Updated segment instance states to use WAL rep states
--Other misc. updates.
--"fixed" flowchart readability issue in g-recovering-a-failed-segment

The overview topics were updated in doc PR https://github.com/greenplum-db/gpdb/pull/5821

TO DO 
--need to add information about tablespaces when available.
--need to test w/ working utilities when available.
--possible changes for replication slots/master failover.

Link to HTML format of updated docs on a GPDB review site.
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-enabling-segment-mirroring.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-enabling-master-mirroring.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-detecting-a-failed-segment.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-checking-for-failed-segments.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-recovering-a-failed-segment.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-recovering-from-segment-failures.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-recovering-a-failed-segment.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-when-a-segment-host-is-not-recoverable.html
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-recovering-a-failed-master.xml
https://docs-msk-gpdb6-dev.cfapps.io/600/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-a-failed-master.xml [hunk 1]
```diff
@@ -4,39 +4,39 @@
...
+          host you are activating. For
+            example:<codeblock>$ gpactivatestandby -d /data/master/gpseg-1</codeblock><p>Where
+              <codeph>-d</codeph> specifies the data directory of the master host you are
+            activating.</p><p>After you activate the standby, it becomes the <i>active</i> or
+              <i>primary</i> master for your Greenplum Database array. </p></li>
+        <li id="ki155816">After the utility completes, run <codeph>gpstate</codeph> with the
+            <codeph>-b</codeph> option to display a summary of the system
+            status:<codeblock>$ gpstate -b</codeblock><p>The master instance status should be
+              <codeph>Active</codeph>. When a standby master is not configured, the command displays
+              <codeph>No master standby configured</codeph>for the standby master status. If you
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-a-failed-master.xml]: Need a space between 'configured</codeph>' and 'for'

File: gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-a-failed-master.xml [hunk 2]
```diff
@@ -4,39 +4,39 @@
...
-        <li id="ki155823">Optional: If you did not specify a new standby host when running the
-            <codeph>gpactivatestandby</codeph> utility, use <codeph>gpinitstandby</codeph> to
-          configure a new standby master at a later time. Run <codeph>gpinitstandby</codeph> on your
-          active master host. For
-          example:<codeblock>$ gpinitstandby -s <i>new_standby_master_hostname</i>
-               </codeblock>
-        </li>
+          example:<codeblock>$ psql <codeph>dbname</codeph> -c 'ANALYZE;'</codeblock></li>
+        <li id="ki155823">Optional: If you not have already done so while activating the prior
+          standby master, you can run <codeph>gpinitstandby</codeph> on the active master host to
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-a-failed-master.xml]: you not have -> you have not

File: gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-from-segment-failures.xml [hunk 3]
```diff
@@ -15,31 +25,36 @@
         </li>
         <li id="ki155645">Troubleshoot the problem that prevents the master host from connecting to
           the segment host. For example, the host machine may need to be restarted or replaced.</li>
-        <li id="ki155646">After the host is online and you can connect to it, run the
-            <codeph>gprecoverseg</codeph> utility from the master host to reactivate the failed
-          segment instances. For example:<codeblock>$ gprecoverseg</codeblock>
+        <li id="ki155646">After the host is online and you can connect to it, run the <codeph><xref
+              href="../../../utility_guide/admin_utilities/gprecoverseg.xml"
+            >gprecoverseg</xref></codeph> utility from the master host to reactivate the failed
+          segment instances and starts the process of sycnronizing the master and mirror instances.
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-from-segment-failures.xml]: starts the process of sycnronizing -> start the process of syncronizing

File: gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-from-segment-failures.xml [hunk 4]
```diff
@@ -62,37 +77,47 @@
...
-          <codeph>gprecoverseg</codeph>:<codeblock>$ gprecoverseg</codeblock>
-        </li>
+        <li>Troubleshoot the problem that caused the double fault and ensure that the segment hosts
+          are operational and are accessible from the master host.</li>
+        <li id="ki165670">Restart Greenplum Database. The <codeph><xref
+              href="../../../utility_guide/admin_utilities/gpstop.xml" type="topic" format="dita"
+              >gpstop</xref></codeph> option <codeph>-r</codeph> stops and restarts the
+          system.<codeblock>$ gpstop -r</codeblock></li>
+        <li id="ki165671">After the system restarts, run <codeph>gprecoverseg</codeph> reactivate
+          the failed segment instances.<codeblock>$ gprecoverseg</codeblock>
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-recovering-from-segment-failures.xml]: add 'to' before reactivate

File: gpdb-doc/dita/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.xml [hunk 5]
```diff
@@ -8,59 +8,69 @@
...
-          <p>You can remove the backup directory once the standby is successfully configured.</p>
-        </li>
+            <codeph>backup_gpseg-1</codeph>:<codeblock>$ mv /data/master/gpseg-1 /data/master/backup_gpseg-1</codeblock><p>You
+            can remove the backup directory once the standby is successfully configured.</p></li>
         <li id="ki160940">Initialize a standby master on the original master host. For example, run
           this command from the current master host,
-          smdw:<codeblock>$ gpinitstandby -s mdw</codeblock>
-        </li>
+          smdw:<codeblock>$ gpinitstandby -s mdw</codeblock></li>
         <li>After the initialization completes, check the status of standby master, mdw, run
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.xml]: I think a new sentence should start at "Run gpstate..." or something else to fix it.

File: gpdb-doc/dita/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.xml [hunk 6]
```diff
@@ -8,59 +8,69 @@
...
-            <codeph>gpstate</codeph> with the <codeph>-f </codeph> option to check the status:<codeblock>$ gpstate -f</codeblock>
-          <p>The status should be <i>In Synch</i>.</p>
-        </li>
+              <codeph><xref href="../../../utility_guide/admin_utilities/gpstate.xml#topic1"
+              type="topic" format="dita"/></codeph> with the <codeph>-f</codeph> option to check the
+          standby master status:<codeblock>$ gpstate -f</codeblock> The standby master status should
+          be <codeph>passive</codeph>, and the WAL sender state should be
+          <codeph>streaming</codeph>.</li>
         <li>Stop Greenplum Database master instance on the standby master. For
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.xml]: Stop ^the^ Greenplum Database master..

File: gpdb-doc/dita/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.xml [hunk 7]
```diff
@@ -8,59 +8,69 @@
...
-        </li>
+          master host, mdw, that is currently a standby master. For
+            example:<codeblock>$ gpactivatestandby -d $MASTER_DATA_DIRECTORY</codeblock><p>Where the
+              <codeph>-d</codeph> option specifies the data directory of the host you are
+            activating.</p></li>
+        <li id="ki165618">After the utility completes, run <codeph>gpstate</codeph> with the
+            <codeph>-b</codeph> option to display a summary of the system
+            status:<codeblock>$ gpstate -b</codeblock><p>The master instance status should be
+              <codeph>Active</codeph>. When a standby master is not configured, the command displays
+              <codeph>No master standby configured</codeph>for the standby master state.</p></li>
```
Comment (litzell) [inline gpdb-doc/dita/admin_guide/highavail/topics/g-restoring-master-mirroring-after-a-recovery.xml]: Need a space between configured and for

**Comment by mkiyama** [comment]
@skahler-pivotal 
This PR is to remove the filerep stuff and prep for the WAL rep changes.

We're waiting for the WAL rep and replication slots changes to update the docs
unless you or somebody knows what the changes are already.

About "contact Pivotal Support". That is conditional text that should only show up in the enterprise version of the docs. The text should be ignored (not published) when the OSS docs are generated.

Due to resource limitations we are single-sourcing from the OSS docs.

let me know if you have questions, or answers.

**Pull request merged**